guide/phone_order_delivery_service/phone_order_delivery_service.bal

```diff
@@ -0,0 +1,218 @@
+//}
+
+//@kubernetes:Deployment {
+//image:"ballerina.guides.io/phone_store_service:v1.0",
+//name:"ballerina-guides-phone_order_delivery_service"
+//}
+
+//endpoint http:Listener listener {
+//port:9091
+//};
+
+//@http:ServiceConfig {basePath:"/phonestore1"}
+//service<http:Service> phone_store_service bind listener {
+
+//Deploying on docker
+
+// import ballerinax/docker;
+// Other imports
+
+// Type definition for a  Deliver order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@docker:Config {
+//registry:"ballerina.guides.io",
+//name:"phone_order_delivery_service",
+//tag:"v1.0"
+//}
+
+// Service endpoint
+//@docker:Expose{}
+//endpoint http:Listener listener {
+//port:9091
+//};
+
+// phone store service, which allows users to order phones online for delivery
+//@http:ServiceConfig {basePath:"/phonestore1"}
+//service<http:Service> phone_order_delivery_service bind listener {
+
+type phoneDeliver record {
+    string customerName;
+    string address;
+    string contactNumber;
+    string deliveryPhoneName;
+};
+
+json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+jms:Connection jmsConnection = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a queue sender using the created session
+endpoint jms:QueueSender jmsProducer2 {
+    session:jmsSession,
+    queueName:"DeliveryQueue"
+};
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(jmsConnection, {
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Service endpoint
+endpoint http:Listener deliveryEP {
+    port:9091
+};
+
+@http:ServiceConfig {basePath:"/deliveryDetails"}
+
+// phone store service, which allows users to order phones online for delivery
+service<http:Service> phone_order_delivery_service bind deliveryEP {
+    // Resource that allows users to place an order for a phone
+    @http:ResourceConfig { consumes: ["application/json"],
+        produces: ["application/json"] }
+
+    sendDelivery(endpoint caller, http:Request enrichedreq) {
+        http:Response response;
+        phoneDeliver newDeliver;
+        json reqPayload;
+
+        log:printInfo(" Order Details have received from phone_store_service");
+
+        // Try parsing the JSON payload from the request
+        match  enrichedreq.getJsonPayload() {
+            // Valid JSON payload
+            json payload => reqPayload = payload;
+            // NOT a valid JSON payload
+            any => {
+                response.statusCode = 400;
+                response.setJsonPayload({"Message":"Invalid payload - Not a valid JSON payload"});
+                _ = caller -> respond(response);
```

Comment:
Shall we handle the possible error here, similar to https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/hello-world-service/hello_world_service.bal#L14 ? 

Also we don't use spaces around `->` when it's an endpoint action. Shall we change to `caller->respond(response)`?

Reply:
done